### DISCUSSION TITLE
thanhle.blog/fe-framework-review-svelte-kit

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#78] [Ideas]
# thanhle.blog/fe-framework-review-svelte-kit

Svelte này đỉnh lắm, đọc thử xem

https://thanhle.blog/blog/fe-framework-review-svelte-kit

**Comment by thanhlmm** [comment]
https://millionjs.org/docs/start-here

Một option đang beta dưới dạng: Interface React, nhưng engine bên dưới build giống Svelte
